doc/example/getting_started.ipynb

```diff
@@ -661,6 +680,26 @@
     "ax.set_xlabel(\"Log10(Parameter value)\");"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Parameter profiles need to be computed until likelihood ratio is small enough to compute confidence intervals with selected confidence level (see `ratio_min` attribute of `ProfileOptions`). It is also possible to visualze nested confince intervals correspondig to different confidence levels"
```

Comment:
```suggestion
    "Parameter profiles need to be computed until the likelihood ratio is small enough to compute confidence intervals with the selected confidence level (see `ratio_min` attribute of `ProfileOptions`). It is also possible to visualize nested confidence intervals corresponding to different confidence levels."
```

Please move the legend to reduce overlap: 
<img width="1686" height="1298" alt="image" src="https://github.com/user-attachments/assets/167fb72e-e804-46eb-8380-53bf75b897f8" />

Reply:
How about at the top of the plot, as many CIs might get close to the boundaries and `loc='best'` didn't work?
<img width="697" height="574" alt="image" src="https://github.com/user-attachments/assets/14943682-3d74-4f39-8a21-96ec3e80294f" />